debian/changelog

```diff
@@ -1,3 +1,9 @@
+android-system-core (0.0.10+b1) stretch; urgency=low
```

Comment:
Whenever in-package dependency is involved, we have to update both package version and debian version, or pkg-config won't work correctly.